src/plugins/chart_expressions/expression_xy/public/components/xy_chart.tsx

```diff
@@ -534,6 +547,46 @@ export function XYChart({
     valueLabels !== ValueLabelModes.HIDE &&
     getValueLabelsStyling(shouldRotate);
 
+  const multiClickHandler: ElementClickListener = (values: any) => {
```

Comment:
ElementClickListener has a well defined type, please don't use `(values: any)` because you should have the correct types there

Reply:
Yes I know, I am just trying to figure out which is the type tbh